### DISCUSSION TITLE
PAG最新版AE导出插件，在Windows系统下无法导出带有BMP形式的文件，没有报错提醒，只是会导出失败

### DISCUSSION THREAD

**Discussion by EddieHxc** [Tencent/libpag#2315] [General] [answered]
PAG最新版AE导出插件，在Windows系统下无法导出带有BMP形式的文件，没有报错提醒，只是会导出失败

**Comment by Nico12138-ui** [comment]
请问你解决了吧，我也遇到了一样的问题

**Comment by VomPom** [comment]
+1

**Reply by elevencat** [reply]
+1

**Comment by lvqinsuan** [comment]
求问这个问题有人解决了吗

**Comment by jinwuwu001** [comment]
请问该问题出现的环境是否还在，若还在请帮忙确认如下问题
如下图所示，输入%APPDATA%\H264EncoderTools
<img width="421" alt="1" src="https://github.com/user-attachments/assets/4557a7ea-c694-431a-a15e-49484670cf8f" />

然后回车进入对应文件夹，检查下该文件夹下是否存在名为H264EncoderTools.exe的可执行程序
<img width="710" alt="2" src="https://github.com/user-attachments/assets/67d3015c-c9a6-4b55-a8a5-310ccdf412b7" />

若不存在且此时AE导出bmp文件失败，请在此评论回复下便于确认问题原因

**Comment by jinwuwu001** [comment, marked as answer]
经确认该问题的原因是PAGViewer在进行ae插件安装时没有正确安装264编码工具，该问题是偶现的，目前正在处理中，后续版本会修复，可通过如下两种方式暂时规避该问题
1、可以在PAGViewer中多次安装ae插件，确保%APPDATA%\H264EncoderTools下存在编码工具后再进行pag导出
2、在PAGViewer安装目录下找到可执行程序PAGViewer/H264EncoderTools.exe，将其复制至%APPDATA%\H264EncoderTools，然后再进行pag文件导出
